config/config.go

```diff
@@ -156,6 +156,12 @@ type ExplorerConfig struct {
 // ServersConfig will hold all the confidential settings for servers
 type ServersConfig struct {
 	ElasticSearch ElasticSearchConfig
+	Prometheus    PrometheusConfig
+}
+
+// PrometheusConfig will hold configuration for prometheus, such as the join URL
+type PrometheusConfig struct {
+	PrometheusJoinURL string
```

Comment:
The variable here and in the config should be `PrometheusURL`. We should always use a base url and go from there

Reply:
done